flycheck.el

```diff
@@ -8380,6 +8380,9 @@ See URL `http://jruby.org/'."
   :modes (enh-ruby-mode ruby-mode)
   :next-checkers ((warning . ruby-rubylint)))
 
+(flycheck-def-args-var flycheck-cargo-rustc-args (rust-cargo)
+  :package-version '(flycheck . "0.30"))
```

Comment:
That was quick!  But it's just `30` without the `0.`.  We have dropped the decimal since version 26.